README.rst

```diff
@@ -1,3 +1,12 @@
+**Interested in contributing a code example?** Please let us know about your
+code example by making a new issue to this repository. Thanks!
+
+.. ..
+
```

Comment:
Is the second pair of periods needed?

Reply:
Yes, the second pair of periods are needed. It's a way for me to make a comment in the README.rst without the comment appearing in the rendered version of the readme.

details found in the section on "Comments": https://gist.github.com/dupuy/1855764

Reply:
"In the following comment examples, the reStructuredText comment / directive marker .. is followed by two more periods so that the following blank line does not terminate the comment." I had read that as saying the second pair is needed if you have more than one paragraph (separated by a blank line), which is not the case here. It's not some big deal, I was just curious because I use one pair of periods in RST.

Reply:
Yes, I see what you're saying now. I only need the second pair if I have a new, empty line before the actual comment. Thanks for the clarification, @JoelPasvolsky. I shall remove the second pair of periods and the empty line.